sources/GeneratorSdk/samples/DiaSdk/Microsoft.Dia.Win32Metadata.nuspec

```diff
@@ -17,7 +17,7 @@
     <icon>images\windows.png</icon>
     <repository type="git" url="https://github.com/microsoft/win32metadata.git" />
     <dependencies>
-      <dependency id="Microsoft.Windows.SDK.Win32Metadata" version="54.0.28-preview" />
+      <dependency id="Microsoft.Windows.SDK.Win32Metadata" version="65.0.1-preview-gfdd0306498" />
```

Comment:
This isn't related to ShellExecute, revert?

Reply:
The build keeps changing it. I don't know why. Maybe this file should be .gitignored?

Reply:
It likes you! (Will check it out.)

Reply:
DoAll.ps1 script runs this which scribbles on that file. Should the file be ignored or should this scribbling happen into an obj dir? https://github.com/microsoft/win32metadata/blob/main/scripts/UpdateSampleDependencies.ps1

@mikebattista looks like this came from https://github.com/microsoft/win32metadata/commit/67b8c7e945a884cd0b4ad6adbf9c74b309610a71#diff-48ea1c3e7e64d76be8daf15d44d92b2ea237f91849b86f46c06fce0f803fb7f7 . Any ideas what the solution should be so I don't keep getting this file edited in my local repo? :)

Reply:
Oh, for inner-loop dev, I would add `-ExcludeSamples -ExcludePackages`.